examples/sample-app/cleanup.sh

```diff
@@ -3,8 +3,13 @@
 echo "Killing openshift all-in-one server ..."
 pkill -x openshift
 
+echo "Stopping all k8s docker containers on host ..."
+docker ps | awk 'index($NF,"k8s_")==1 { print $1 }' | xargs -l -r docker stop
```

Comment:
@guangxuli it would still be a small improvement since the current implementation depends on column ordering which, while unlikely to change, could change.  Using the --format approach is more api-based and thus likely to be more robust if docker changes the output of docker ps.